packages/manager/modules/pci/src/projects/project/storages/databases/database/users/add/add.html

```diff
@@ -4,58 +4,47 @@
 >
 </oui-back-button>
 
-<oui-stepper data-on-finish="$ctrl.addUser()">
-    <oui-step-form
-        id="user_details"
-        name="user_details"
-        data-header="{{:: 'pci_databases_users_add' | translate }}"
-        data-prevent-next="true"
-        data-on-cancel="$ctrl.cancel('dashboard::users::add_a_user::cancel')"
-        data-on-submit="$ctrl.trackDatabases('dashboard::users::add_a_user::next_step')"
-        data-editable="!$ctrl.processing"
+<h2>Add user</h2>
```

Comment:
Lot of string are not translated. Is it by requirement?

Reply:
Corrected with https://github.com/ovh/manager/commit/5642e84150fcdb29b4ce1bac79e1f35d7edb8e65